### PR TITLE
Fix #693, Utilize UT macros in sb_UT

### LINKED ISSUE #693: Utilize new macros and correct some sb_UT.c merge conflicts
**Describe the bug**
Doing the merge conflicts for the ic merges, I mis-merged a couple of Joe's changes ( fix #245 ).

**Reporter Info**
Christopher.D.Knight@nasa.gov

### PR DESCRIPTION
**Pull request by CDKnightNASA**
**Describe the contribution**
cleanup of sb_UT.c

Fix #693

**Contributor Info - All information REQUIRED for consideration of pull request**
Christopher.D.Knight@nasa.gov

### CODE REVIEW

File: fsw/cfe-core/unit-test/sb_UT.c [hunk 1]
```diff
@@ -6016,7 +6002,7 @@ void Test_RcvMsg_UnsubResubPath(void)
     SETUP(CFE_SB_Subscribe(MsgId, PipeId));
     ASSERT(CFE_SB_RcvMsg(&PtrToMsg, PipeId, CFE_SB_PEND_FOREVER));
 
-    ASSERT_TRUE(PtrToMsg == NULL);
```
Comment (CDKnightNASA) [inline fsw/cfe-core/unit-test/sb_UT.c]: So this is curious, I need to confirm that the behavior has changed...But I believe the new behavior is correct (this is probably a side-effect of one of the other merges).

File: fsw/cfe-core/unit-test/sb_UT.c [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
-**  GSC-18128-1, "Core Flight Executive Version 6.7"
+**  GSC-18128-1, "Core Flight Executive version 6.7"
```
Comment (skliper) [inline fsw/cfe-core/unit-test/sb_UT.c:2]: Recommend avoiding changes here

**Comment by skliper** [comment]
Recommended commit - `Fix #693, Add MID wrap and minor UT cleanup`

File: fsw/cfe-core/unit-test/sb_UT.c (hunk 2 above)
Comment (CDKnightNASA) [reply fsw/cfe-core/unit-test/sb_UT.c]: GAH, I thought I caught that fat-finger error...I've fixed, squashed, and re-pushed.

**Comment by skliper** [comment]
Failure reported by bundle CI, with OMIT_DEPRECATED=true.

Note - to recreate, from cFS bundle create a branch that references the cfe submodule w/ this commit and push to your fork.  Results can be seen on travis-ci.com.

https://travis-ci.com/github/skliper/cFS/builds/165453917

```
/home/travis/build/skliper/cFS/cfe/fsw/cfe-core/unit-test/sb_UT.c: In function ‘Test_OS_MutSem_ErrLogic’:
/home/travis/build/skliper/cFS/cfe/fsw/cfe-core/unit-test/sb_UT.c:5668:53: error: ‘CFE_OS_SEM_FAILURE’ undeclared (first use in this function); did you mean ‘OS_SEM_FAILURE’?
     UT_SetDeferredRetcode(UT_KEY(OS_MutSemTake), 1, CFE_OS_SEM_FAILURE);
                                                     ^~~~~~~~~~~~~~~~~~
                                                     OS_SEM_FAILURE
/home/travis/build/skliper/cFS/cfe/fsw/cfe-core/unit-test/sb_UT.c:5668:53: note: each undeclared identifier is reported only once for each function it appears in
```

**Comment by CDKnightNASA** [comment]
> Failure reported by bundle CI, with OMIT_DEPRECATED=true.
> 
> Note - to recreate, from cFS bundle create a branch that references the cfe submodule w/ this commit and push to your fork. Results can be seen on travis-ci.com.
> 
> https://travis-ci.com/github/skliper/cFS/builds/165453917
> 
> ```
> /home/travis/build/skliper/cFS/cfe/fsw/cfe-core/unit-test/sb_UT.c: In function ‘Test_OS_MutSem_ErrLogic’:
> /home/travis/build/skliper/cFS/cfe/fsw/cfe-core/unit-test/sb_UT.c:5668:53: error: ‘CFE_OS_SEM_FAILURE’ undeclared (first use in this function); did you mean ‘OS_SEM_FAILURE’?
>      UT_SetDeferredRetcode(UT_KEY(OS_MutSemTake), 1, CFE_OS_SEM_FAILURE);
>                                                      ^~~~~~~~~~~~~~~~~~
>                                                      OS_SEM_FAILURE
> /home/travis/build/skliper/cFS/cfe/fsw/cfe-core/unit-test/sb_UT.c:5668:53: note: each undeclared identifier is reported only once for each function it appears in
> ```

fixed

**Comment by skliper** [comment]
@CDKnightNASA could you link the passing bundle CI?

**Comment by CDKnightNASA** [comment]
> @CDKnightNASA could you link the passing bundle CI?

https://travis-ci.com/github/CDKnightNASA/cFS/jobs/332918553

**Comment by skliper** [comment]
CCB 20200513 - Approved

**Pull request merged**